### PR TITLE
Surveys

### PR DESCRIPTION
**Pull request by UVXY**

### CODE REVIEW

File: screens/Auth/Login.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ export default class SignInScreen extends Component {
 
     goToMain = (userObj) => {
       const navigateAction = NavigationActions.navigate({
-        routeName: "Home",
+        routeName: "Words",
```
Comment (dev-mev) [inline screens/Auth/Login.js]: I'm guessing this was for testing purposes, but it should be switched back to `Home`.

File: screens/WordRoulette.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
+import React, { Component } from 'react'
+import { Image, Linking } from 'react-native';
+import { Container, Header, Content, Card, CardItem, Thumbnail, Text, Button, Icon, Left, Body, Right } from 'native-base';
+import { NavigationActions } from "react-navigation";
+
+export default class WordRoulette extends Component {
```
Comment (dev-mev) [inline screens/WordRoulette.js:6]: Is this tested? It looks like JQuery, is that supported by React Native?

File: screens/WordSearcher.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+   <ScrollView style={styles.container} contentContainerStyle={styles.contentContainer}>
+        
+        <Content>
+          <Card style={{flex: 0}}>
+            <CardItem>
+              <Left>
+                <Thumbnail source={{uri: 'Image URL'}} />
+                <Body>
+                  <Text>Word Search</Text>
+                  <Text note>April 15, 2016</Text>
```
Comment (dev-mev) [inline screens/WordSearcher.js:54]: What is this date?

**Review by dev-mev — CHANGES_REQUESTED** [review]
The stuff in red at the top are left behind from merge conflicts that are unresolved. You have to resolve those or it will break the app. There are some other inline comments with questions. Let me know if you have any questions.

**Comment by UVXY** [comment]
I tried to explain what these things were to you on Saturday.

File: screens/WordSearcher.js (hunk 3 above)
Comment (UVXY) [reply screens/WordSearcher.js:54]: starter file from native base

File: screens/WordRoulette.js (hunk 2 above)
Comment (UVXY) [reply screens/WordRoulette.js:6]: It's for navigating through react. it was used on the initial starter code

File: screens/Auth/Login.js (hunk 1 above)
Comment (UVXY) [reply screens/Auth/Login.js]: It's not for testing purposes. Its the new route to Words

File: screens/WordRoulette.js (hunk 2 above)
Comment (dev-mev) [reply screens/WordRoulette.js:6]: Sorry, I'm talking about all the WordRoulette code that comes next. It looks like JQuery. I haven't tried it, but I didn't think that worked in React Native. I just want to make sure it's working before we merge it.

File: screens/Auth/Login.js (hunk 1 above)
Comment (dev-mev) [reply screens/Auth/Login.js]: Ahh, because that's the page we'll see instead of a homepage. Got it! 👍

File: screens/WordRoulette.js (hunk 2 above)
Comment (UVXY) [reply screens/WordRoulette.js:6]: Oh ya you're right I was supposed to comment that out.

**Review by UVXY — COMMENTED** [review]
Let me know if I missed anything. I commented out the roulette and changed the route back to Home

**Review by dev-mev — APPROVED** [review]

**Review by dev-mev — CHANGES_REQUESTED** [review]
I'm not seeing the changes. Did you push them?

**Pull request closed without merging**